tekton/resources/nightly-tests/dashboard-deploy-test-ppc64le-template.yaml

```diff
@@ -0,0 +1,201 @@
+# Copyright 2021 The Tekton Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: tekton-dashboard-nightly-test-ppc64le
+spec:
+  params:
+  - name: containerRegistry
+  - name: targetArch
+  - name: namespace
+  - name: remoteHost
+  - name: remotePort
+  - name: remoteUser
+  - name: remoteSecret
+  resourcetemplates:
+  - apiVersion: tekton.dev/v1beta1
+    kind: PipelineRun
+    metadata:
+      generateName: tekton-dashboard-$(tt.params.targetArch)-nightly-run-
+      namespace: $(tt.params.namespace)
+    spec:
+      timeout: 2h
+      workspaces:
+      # this workspace will be used to share info between tasks
+      - name: shared-workspace
+        volumeClaimTemplate:
+          spec:
+            accessModes:
+              - ReadWriteOnce
+            resources:
+              requests:
+                storage: 4Gi
+      # this workspace will be used to store ssh key
+      - name: ssh-secret
+        secret:
+          secretName: $(tt.params.remoteSecret)
+          items:
+          - key: privatekey
+            path: id_rsa
+            # yamllint disable rule:octal-values
+            mode: 0600
+            # yamllint enable
+      # this workspace will be used to store registry credentials
+      - name: registry-credentials
+        secret:
+          secretName: registry-credentials
+      # this workspace will be used to store registry self-signed certificate
+      - name: registry-certificate
+        secret:
+          secretName: registry-certificate
+      pipelineSpec:
+        workspaces:
+        - name: shared-workspace
+        - name: ssh-secret
+        - name: registry-credentials
+        - name: registry-certificate
+        params:
+        - name: container-registry
+        - name: target-arch
+        - name: remote-host
+        - name: remote-port
+        - name: remote-user
+        tasks:
+        - name: git-clone-dashboard
+          taskRef:
+            name: git-clone
+          params:
+          - name: url
+            value: https://github.com/tektoncd/dashboard
+          - name: revision
+            value: main
+          - name: subdirectory
+            value: src/github.com/tektoncd/dashboard
+          workspaces:
+          - name: output
+            workspace: shared-workspace
+            subPath: source-code
+        - name: create-k8s-cluster
```

Comment:
Is this going to be the way for the other tests too eventually? 
Is the command that create the cluster blocking until ready, or how do we know that the k8s cluster is ready for serving?

Reply:
> Is this going to be the way for the other tests too eventually?

Yes. All other tests will have same approach. 

> Is the command that create the cluster blocking until ready

Yes, it is a blocking command. It takes around 5 minutes to setup a cluster.